src/hubbleds/pages/01-spectra-&-velocity/__init__.py

```diff
@@ -56,6 +58,17 @@ def _on_galaxy_table_row_selected(row):
 
 @solara.component
 def Page():
+
+    def glue_setup():
+        gjapp = JupyterApplication(GLOBAL_STATE.data_collection, GLOBAL_STATE.session)
+        dotplot_test_data = Data(x=[randint(1, 10) for _ in range(30)])
+        dotplot_test_data.style.color = "black"
+        gjapp.data_collection.append(dotplot_test_data)
+        dotplot = gjapp.new_data_viewer(HubbleDotPlotView, data=dotplot_test_data, show=False)
+        return gjapp, dotplot
+
+    gjapp, dotplot = solara.use_memo(glue_setup, [])
```

Comment:
This is pretty neat. I'm guessing it's the way glue-solara stuff does it?

Reply:
Yeah, this ended up being the key piece from `glue-solara` to get things working.